cfme/tests/webui/test_general_ui.py

```diff
@@ -339,6 +341,42 @@ def test_tls_openssl_verify_mode(temp_appliance_preconfig, request):
     assert appliance.advanced_settings["smtp"]["openssl_verify_mode"] == "none"
 
 
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(blockers=[BZ(1741310)], automates=[1741310])
+@pytest.mark.parametrize("destination", ["AnsibleTowerExplorer", "ConfigManagement"])
+def test_add_provider_button_accordion(appliance, destination):
+    """
+    Test that add_provider button is visible after clicking accordion on
```

Comment:
Is this the button that's only displayed when there are no providers configured? I think you'll want to use `has_no_providers` fixture if so, to ensure the appliance is 'clean' when this runs.

Reply:
Does the `has_no_providers` fixture work for these extra providers? Ansible Tower and Config Manager -> Satellite?

Reply:
Nope, `has_no_providers` won't work here. We'll have to write a separate fixture for it.

Reply:
**Required:**
I think writing a local fixture to remove these providers before the test starts would do for now, what do you think @john-dupuy  @mshriver ? Writing a cleanup function for ConfigManager providers like the one we have for BaseProvider seems a little out of scope of this PR and a little futile to me.

Reply:
Thanks @valaparthvi, let me look into this. Perhaps it will be easy to implement a fixture to clear the providers. I am also working on #9317, but haven't got very far yet, maybe this could be something that'd be easy to implement as a class method on `ConfigManager`.

Reply:
@valaparthvi now that #9317, we can finally use `has_no_providers` with Config Managers :tada:

Reply:
#9317 got merged! That was quick :grinning:
@john-dupuy we already have a test [`test_add_button_on_provider_all_page`](https://github.com/ManageIQ/integration_tests/blob/61a7eb1f82ebce57186b3591b26901cf804fd1d1/cfme/tests/webui/test_general_ui.py#L275), can you see if we can merge this test into it?

Reply:
Ah, okay, let me look into it.

Reply:
@valaparthvi the test case you linked is a bit different. Here according to the BZ the add provider button disappears if you click the somewhere in the accordion and then click back. That test case just looks like it's checking that the button shows up when there's no provider.